tests/conftest.py

```diff
@@ -85,16 +84,25 @@ def n_subsystems(request):
     return request.param
 
 
-# Device specification:
+# Device specification
 # The device name will be provided by the binaries.
-device_name = backend_info()["NAME"]
+# Checking binaries:
+try:
+    from pennylane_lightning.lightning_qubit_ops import backend_info
+except ModuleNotFoundError:
+    try:
+        from pennylane_lightning.lightning_kokkos_ops import backend_info
+    except ModuleNotFoundError:
+        # with no binaries we fallback to LightningQubit, and this one will fallback to default.
+        from pennylane_lightning import LightningQubit as LightningDevice
+    else:
```

Comment:
Fine for now, we but can think about a more elegant solution to this later.

Reply:
Yes. We can make it better later.